### PR TITLE
Release 2.0.4

### PR DESCRIPTION
**Pull request by renatho**
- Update changelog.
- Bump version.
- Bump tested WP versions.
- Update pot file.

### CODE REVIEW

**Comment by renatho** [comment]
I'm wondering about this release since it's just to bump the tested versions. Maybe we could just update the fields in the SVN directly. WDYT?

File: changelog.txt [hunk 1]
```diff
@@ -1,5 +1,9 @@
 *** Changelog ***
 
+2021.08.22 - version 2.0.4
+
+* Bump tested WP versions - #106
```
Comment (donnapep) [inline changelog.txt:5]: I don't think we need to update the change log since there are no changes to functionality.

**Comment by donnapep** [comment]
> I'm wondering about this release since it's just to bump the tested versions. Maybe we could just update the fields in the SVN directly. WDYT?

@renatho Since we're just updating the tested up to versions, we shouldn't bump the plugin version number or update the .pot file. This will ensure that a new plugin version is not pushed out to new sites. We really only need to update the readme.

**Comment by renatho** [comment]
As discussed, closed in favor of https://github.com/woocommerce/sensei-media-attachments/pull/107

**Pull request closed without merging**